src/logic/operate.js

```diff
@@ -0,0 +1,37 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+
+  if (operation === '÷' && numberTwo === '0') {
+    alert('Can\'t divide by zero');

```

Comment:
Nice attempt here to catch division by zero. However, this may not be the best way to implement this. We have to use React's full capabilities to handle errors.

In the `App` component we can include an `error` state. Then in our `calculate.js` file we can use a `try` and `catch` block where in the `catch` part we will return an error whenever there's a division by zero detected by the app.

Now then, since we can have an error state of either `null` or something we can use this state to pass from our `App` down to `Display` component to display the appropriate error message on the screen.